### ISSUE TITLE
Bug: Database issue with Zeit

### ISSUE THREAD

**Issue by jferragut** [jferragut/flask-microservices-boilerplate#2]
**Bug**: Zeit doesn't appear to support MySQL databases or any for the time being. (which breaks the example) 

<img width="949" alt="Screen Shot 2019-08-25 at 1 49 36 AM" src="https://user-images.githubusercontent.com/31458614/63646101-b39b9980-c6da-11e9-8bf8-84433b517fe5.png">

**Request**: Need to set up a firebase example so that there is some sort of queryable version of an endpoint.

**Comment by alesanchezr** [comment]
Zeit v1.0 supported it, but the new approach is to ahve your databases on a different server like JawDB, AWS, Google Gloud or something like that.

The cheapest one I've seen is jawsdb but those 3 are great.

**Comment by jferragut** [comment]
I agree to this approach @alesanchezr . Commit and PR incoming to set up with firestore.

**Comment by jferragut** [comment]
PR waiting for approval

**Comment by jferragut** [comment]
**Todo:**
- [ ] update documentation to include explanation of how to setup env variables with zeit
- [ ] update docs with explanation on where to get data for key.json
